File: Backend/Blog.Core/Entities/Blog.cs
namespace Blog.Core.Entities;

public class Blog
{
    public Guid Id { get; set; }
    public required string Name { get; set; }
    public required string Description { get; set; }
    public required string HtmlContent { get; set; }
    public DateTime Created { get; set; }
    public Guid AuthorId { get; set; }
    public User? Author { get; set; }
}

File: Frontend/src/entities/model/blog.ts
import type {Guid} from "guid-typescript";
import type {User} from "./user.ts";

export type Blog = {
    id: Guid;
    name: string;
    description: string;
    markdownContent: string;
    created: Date;
    authorId: Guid;
    author?: User;
}